src/configs/r.rs

```diff
@@ -0,0 +1,22 @@
+use crate::config::{ModuleConfig, RootModuleConfig};
+
+use starship_module_config_derive::ModuleConfig;
+
+#[derive(Clone, ModuleConfig)]
+pub struct RConfig<'a> {
+    pub format: &'a str,
+    pub style: &'a str,
+    pub symbol: &'a str,
+    pub disabled: bool,
+}
+
+impl<'a> RootModuleConfig<'a> for RConfig<'a> {
+    fn new() -> Self {
```

Comment:
This is implemented like this now:
```suggestion
impl<'a> Default for RConfig<'a> {
    fn default() -> Self {
```

Also can please you add the module to `FullConfig` in `configs/mod.rs`?

Reply:
Ok my mistake, I just realized I didn't merge the starship repository with my fork and got out of sync. Thanks for the quick response. Let me take another look to the PR with latest changes then. 👍